docker/Dockerfile.frontend

```diff
@@ -1,11 +1,15 @@
-FROM node:16.13.0 AS frontend
+FROM node:16.13.0-slim AS frontend
 
 WORKDIR /app/oppia
 
-RUN apt-get update -y && apt-get upgrade -y \
+RUN apt-get update -y && apt-get install -y --no-install-recommends \
```

Comment:
Are any of these packages already present in the image? If so, then we no longer upgrade them

Reply:
git , curl , chromium and ca-certificate package is not present in node:16.13.0-slim , so we have to download it manually.